examples/twitter_support/main.py

```diff
@@ -0,0 +1,98 @@
+    input_tweet_2 = "Keira Townsend: I hate the new design of the iphone. It is so ugly. I am switching to Samsung"
+    assert classify_and_respond(input_tweet_2) == Response(requires_ticket=False, response="Hi, we are sorry to hear that. We will take this into consideration and let the product team know of the feedback")
+    input_tweet_3 = "Thomas Bell: @Amazonsupport. I have a question about ordering, do you deliver to Finland?"
+    assert classify_and_respond(input_tweet_3) == Response(requires_ticket=True, response="Hi, thanks for reaching out. The question will be sent to our support team and they will get back to you as soon as possible")
+    input_tweet_4 = "Jillian Murphy: Just bought the new goodybox and so far I'm loving it!"
+    assert classify_and_respond(input_tweet_4) == Response(requires_ticket=False, response="Hi, thanks for reaching out. We are happy to hear that you are enjoying the product")
+
+@monkey.patch
+def create_support_ticket(tweet_text: str) -> SupportTicket:
+    """
+    Using the tweet text create a support ticket for saving to the internal database
+    """
+
+@monkey.align
+def align_supportticket():
+    input_tweet_1 = "Laia Johnson: I really like the new shovel but the handle broke after 2 days of use. Can I get a replacement?"
+    assert create_support_ticket(input_tweet_1) == SupportTicket(name = "Laia Johnson", issue="Needs a replacement product because the handle broke", urgency = "high")
+    input_tweet_2 = "Thomas Bell: @Amazonsupport. I have a question about ordering, do you deliver to Finland?"
+    assert create_support_ticket(input_tweet_2) == SupportTicket(name="Thomas Bell", issue="Answer whether we deliver to Finland", urgency="low")
+    input_tweet_3 = "Jillian Murphy: Just bought the new goodybox and so far I'm loving it! The cream package was slightly damaged however, would need that to be replaced"
+    assert classify_and_respond(input_tweet_3) == SupportTicket(name="Jillian Murphy", issue="Needs a new cream as package was slightly damaged", urgency="medium")
+
+def main():
+    """
+    Run through the workflow of twitter support bot.
+    Example usecase uses 3 tweets, where first two require a support ticket and final one does not 
+    """
+    # start with calling aligns
+    align_respond()
+    align_supportticket()
+
+    input_tweet_1 = "Jack Bell: Bro @Argos why did my order not arrive? I ordered it 2 weeks ago. Horrible service"
+    response = classify_and_respond(input_tweet_1)
+    # requires_ticket=True 
+    # response="Hi Jack, we're really sorry to hear about this. We'll look into it right away and get back to you as soon as possible. Could you please provide us with your order number?"
+    
+    if response.requires_ticket:
+        ticket = create_support_ticket(input_tweet_1)
+        # name='Jack Bell' 
```

Comment:
Can you please clean up the remnant comments?

Reply:
The remnant comments actually are showing what the output is for that patch function, so they are left deliberately for people to get a quick sense of input-output pairs for new datapoints